jscomp/gentype/GenTypeCommon.ml

```diff
@@ -2,6 +2,23 @@ module StringMap = Map.Make (String)
 module StringSet = Set.Make (String)
 module Config = GenTypeConfig
 
+module DocString : sig
+  type t
+  val make : string option -> t
```

Comment:
Do we need this abstraction?
An explicit string option would remove make and the interface.
Unless this will change in future.

Reply:
We can probably do without it. It was mostly a way to make sense of all the places where we had docstrings, where they would occasionally be formatted as comments already, and occasionally not. But I can probably just make it a string option now again. Let me give it a shot.